input/resources/CareCommunication_Ex_receive_Tek-G_new_replacementCPR.json

```diff
@@ -0,0 +1,260 @@
+      },
+      "eventCoding" : {
+        "system" : "http://medcomfhir.dk/ig/terminology/CodeSystem/medcom-messaging-eventCodes",
+        "code" : "care-communication-message"
+      },
+      "destination" : [{
+        "extension" : [{
+          "url" : "http://medcomfhir.dk/ig/messaging/StructureDefinition/medcom-messaging-destinationUseExtension",
+          "valueCoding" : {
+            "system" : "http://medcomfhir.dk/ig/terminology/CodeSystem/medcom-messaging-destinationUse",
+            "code" : "primary"
+          }
+        }],
+        "endpoint" : "https://sor2.sum.dsdn.dk/#id=441211000016000",
+        "receiver" : {
+          "reference" : "Organization/9cf4942a-b0ed-45ba-b13d-6a655f2c0acf"
+        }
+      }],
+      "sender" : {
+        "reference" : "Organization/f9342201-81bd-4a41-a7e5-0f2ae8823656"
+      },
+      "source" : {
+        "endpoint" : "https://sor2.sum.dsdn.dk/#id=1339531000016004"
+      },
+      "focus" : [{
+        "reference" : "Communication/f1947b33-8b35-4cdf-9e0a-0141352f8e42"
+      }],
+      "definition" : "http://medcomfhir.dk/ig/carecommunication/medcom-careCommunication-message-definition|4.0.0"
+    }
+  },
+  {
+    "fullUrl" : "https://medcomfhir.dk/ig/carecommunication/Patient/442804e5-2f1e-4a73-b1b9-a3a0eaff1849",
+    "resource" : {
+      "resourceType" : "Patient",
+      "id" : "442804e5-2f1e-4a73-b1b9-a3a0eaff1849",
+      "meta" : {
+        "profile" : ["http://medcomfhir.dk/ig/core/StructureDefinition/medcom-core-patient"]
+      },
+      "text" : {
+        "status" : "generated",
+        "div" : "<div xmlns=\"http://www.w3.org/1999/xhtml\"><a name=\"Patient_442804e5-2f1e-4a73-b1b9-a3a0eaff1849\"> </a><p class=\"res-header-id\"><b>Generated Narrative: Patient 442804e5-2f1e-4a73-b1b9-a3a0eaff1849</b></p><a name=\"442804e5-2f1e-4a73-b1b9-a3a0eaff1849\"> </a><a name=\"hc442804e5-2f1e-4a73-b1b9-a3a0eaff1849\"> </a><a name=\"442804e5-2f1e-4a73-b1b9-a3a0eaff1849-da-DK\"> </a><p style=\"border: 1px #661aff solid; background-color: #e6e6ff; padding: 10px;\">Peter null (official) Male, DoB Unknown ( urn:oid:1.2.208.176.1.6.1.3.177#1206550VK9)</p><hr/></div>"
+      },
+      "identifier" : [{
+        "system" : "urn:oid:1.2.208.176.1.6.1.3.177",
```

Comment:
Hvad er dette for et system? Jeg kan ikke lige finde det :)

Reply:
Kommer fra det ValueSet tilknyttet dette Identifier.system: https://hl7.dk/fhir/core/StructureDefinition-dk-core-d-ecpr-identifier.html